src/lib/bootloader/grub2_widgets.rb

```diff
@@ -336,6 +338,19 @@ def init
     def store
       grub2.secure_boot = value
     end
+
+    def validate
+      return true if Yast::Mode.config ||
+        !Yast::Arch.s390 ||
+        value == Systeminfo.secure_boot_active?
```

Comment:
NP: I find a bit more readable to have it each on separate line and separate return as it is not related. So something like
```suggestion
return true if Yast::Mode.config
return true unless Yast::Arch.s390
return true if value == Systeminfo.secure_boot_active?
```

Reply:
I personally wouldn't have formatted it this way. But this is what our rubocop rules allow/enforce. To completely reformat it might even run into some complexity check failures.